src/normaliz.cc

```diff
@@ -804,8 +880,9 @@ static Obj Func_NmzConeProperty(Obj self, Obj cone, Obj prop)
 {
     if (!IS_CONE(cone))
         ErrorQuit("<cone> must be a Normaliz cone", 0, 0);
-    if (!IS_STRING_REP(prop))
-        ErrorQuit("<prop> must be a string", 0, 0);
+    if (!IS_STRING_REP(prop) && !IS_INT(prop))
+        ErrorQuit("<prop> must be a string or a value from NMZConeProperties",
```

Comment:
There is no `NMZConeProperties`; perhaps you mean `_NmzRecordOfConeProperties`? In any case, since you only check for an integer, I'd say that:
```suggestion
        ErrorQuit("<prop> must be a string or an integer",
```
Even better would be to check for a small non-negative integer. You can use the `IS_NONNEG_INTOBJ` for that.